packages/editor/src/components/document-bar/index.js

```diff
@@ -52,21 +41,21 @@ const MotionButton = motion( Button );
  * ```jsx
  * <DocumentBar />
  * ```
- * @param {Object}                                   props       The component props.
- * @param {string}                                   props.title A title for the document, defaulting to the document or
- *                                                               template title currently being edited.
- * @param {import("@wordpress/components").IconType} props.icon  An icon for the document, defaulting to an icon for document
```

Comment:
"import("@wordpress/components")" is currently output [in the documentation](https://github.com/WordPress/gutenberg/tree/trunk/packages/editor#documentbar), so I've changed the param type here to just `IconType`.

I'm not sure what a good way is to indicate the type comes from `@wordpress/components`.

Reply:
If we still want to indicate that it comes from `@wordpress/components` within the file, but not in the docs, could we try something like the following slightly further up the file?

```
/** @typedef {import("@wordpress/components").IconType IconType */
```

I don't know how legitimate that is, but just looking at a couple of examples elsewhere that appear to do that:

https://github.com/WordPress/gutenberg/blob/3ea2d42b0a6a206663735a47f9796bd42eda2186/packages/blocks/src/api/parser/serialize-raw-block.js#L11

and

https://github.com/WordPress/gutenberg/blob/c56672504bada434aee1ec331366cd2a10ab2b38/packages/block-editor/src/components/list-view/index.js#L69-L70

Reply:
Done in https://github.com/WordPress/gutenberg/pull/65170/commits/7997eb3e80563e44e034f9d1d946a4bfab7d7ba2